### PR TITLE
fixed bug causing crash in pdf

### PR DESCRIPTION
**Pull request by bianchini**
fixes a bug

### CODE REVIEW

**Comment by bianchini** [comment]
wait a second... before merging, I have another fix

**Comment by bianchini** [comment]
Ok, it should be protected now

File: VHbbAnalysis/Heppy/python/VHGeneratorAnalyzer.py [hunk 1]
```diff
@@ -185,7 +185,7 @@ def makeMCInfo(self, event):
                     LHE_weights_scale.append(w)
             # thse are up/down variations for the NNPDF: https://twiki.cern.ch/twiki/bin/view/CMS/TTbarHbbRun2ReferenceAnalysisLimits
             if wid in [2005,2067]:
-                print  w.wgt
```
Comment (arizzi) [inline VHbbAnalysis/Heppy/python/VHGeneratorAnalyzer.py]: this I've already fixed locally

Comment (bianchini) [inline VHbbAnalysis/Heppy/python/VHGeneratorAnalyzer.py]: Cnan you fix it or you need a new commit?

**Pull request merged**